test/query/personregisterHooks.test.tsx

```diff
@@ -15,7 +15,7 @@ import { expect } from 'chai';
 describe('personregisterHooks tests', () => {
   const queryClient = new QueryClient();
 
-  it('loads personregister correctly', async () => {
+  xit('loads personregister correctly', async () => {
```

Comment:
Gir dette en warning eller noe? Det er fare for at den bare blir sånn for alltid, siden den aldri vil feile 😅

Reply:
Testen vil stå som `pending` under kjøring av testene så forhåpentligvis plukker vi den opp :sweat_smile: